packages/sdk-rtl/src/apiMethods.ts

```diff
@@ -26,17 +26,169 @@
 
 import { Readable } from 'readable-stream'
 import {
-  SDKResponse,
+  agentPrefix,
+  Authenticator,
   HttpMethod,
   ITransportSettings,
-  Values,
   sdkOk,
-  agentPrefix,
-  Authenticator,
+  SDKResponse,
+  Values,
 } from './transport'
 import { IAuthSession } from './authSession'
 
-export class APIMethods {
+/**
+ * Creates an "sdk" to be used with the Typescript SDK's funcs.ts file
+ * @param authSession authentication session
+ * @param apiVersion version of API to use (e.g. "3.1" or "4.0")
+ * @param sdkVersion Looker release version (e.g. "21.4")
+ */
+export const functionalSdk = (
```

Comment:
See my comment on the demo. Can we wrap this method in methods added to the SDK package. The SDK function can hard code the apiVersion and sdkVersion.